root/search/components/EventResults.js

```diff
@@ -41,6 +42,10 @@ function buildResult(result, index) {
       </td>
       <td>
         <ArtistRoles relations={event.performers} />
+        {manifest.js(
+          'common/components/ArtistRoles',
+          {async: 'async'},
+        )}
```

Comment:
These should only get included once per page (so probably output it with the table instead of every cell).